airflow/bin/cli.py

```diff
@@ -383,52 +621,202 @@ def clear(args):
         end_date=args.end_date,
         only_failed=args.only_failed,
         only_running=args.only_running,
-        confirm_prompt=not args.no_confirm)
+        confirm_prompt=not args.no_confirm,
+        include_subdags=not args.exclude_subdags)
+
+
+def restart_workers(gunicorn_master_proc, num_workers_expected):
```

Comment:
@abloomston I think you were you needed these changes for this ticket, right? https://cloverhealth.atlassian.net/browse/AIR-1